### ISSUE TITLE
Doesn't open on click

### ISSUE THREAD

**Issue by FezVrasta** [airbnb/react-dates#126]
So... I'm using it in this way:

```
import '!style!css!sass!react-dates/css/variables.scss';
import '!style!css!sass!react-dates/css/styles.scss';

function Test() {
  return (
   <DateRangePicker />
  );
}
```

It get properly rendered but when I click on the fields, the calendars are not shown...why?

**Comment by majapw** [comment]
Hi @FezVrasta! The `DateRangePicker` is a fully controlled component which means that you need to create a wrapper component that maintains the focused element and selected date in its state and updates those values using the provided `onFocusChange` and `onDatesChange` callbacks. You can see an example of the code you'd need for this kind of wrapper component here: https://github.com/airbnb/react-dates/blob/master/examples/DateRangePickerWrapper.jsx

Basically, without this setup, no interactions will work.

**Comment by FezVrasta** [comment]
Thanks!

**Comment by mattgabor** [comment]
I'm running into the same issue, properly rendered but no response on click, but here's how my component setup looks:

`            <DateRangePicker
              startDate={null}
              endDate={null}
              focusedInput={focusedInput}
              onDatesChange={this.onDatesChange}
              onFocusChange={this.onFocusChange}
            />
`

The onDatesChange and onFocusChange functions are identical to the wrapper, they simply call setState, and focusedInput is set to null in the constructor

**Comment by realisation** [comment]
I have a similar problem where I have provided the required props, but on click nothing renders. Do I still need a wrapper?

**Comment by erin-doyle** [comment]
@realisation can yo provide some code to show?  Otherwise it's hard to say what the problem may be with your approach.

**Comment by realisation** [comment]
I was simply using it without any wrapper component in my jsx. Inside an
enclosing parent div, I had two single date pickers with a span between
them. But then I made a wrapper component doing a singledatepicker alone,
by itself, in a parent div and it works just fine.
